pipeline_tools/http_requests.py

```diff
@@ -238,9 +245,10 @@ def _http_request(self, *args, **kwargs):
     @staticmethod
     def _get_method(http_method):
         session = requests.Session()
-        # The default max_redirects is 30. We need to boost it because DSS sometimes redirects us more than 30 times,
-        # which causes an error if we don't allow for more.
-        session.max_redirects = 100
+        retry_policy = RetryPolicy(read=5, status=5, status_forcelist=frozenset({500, 502, 503, 504}))
```

Comment:
I'm not sure we want to retry 5xx errors in this way since we would normally be retrying those with tenacity. We probably want to keep retrying all errors in tenacity and only use urllib3 retry mechanism to follow redirects.

Reply:
That makes sense to me!